src/styles/blocks/card.css

```diff
@@ -59,18 +61,21 @@
 
 .card__button {
   display: block;
-  background-color: var(--blue-color);
-  border: 1px solid transparent;
-  border-radius: 5px;
-  color: #fff;
+  padding: 11px 70px;
+
   font-weight: 700;
   font-size: 14px;
-  text-align: center;
+  line-height: 16px;
   text-decoration: none;
   text-transform: uppercase;
-  width: 166px;
-  height: 40px;
-  line-height: 40px;
+
+  color: #fff;
+  background-color: var(--blue-color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  transition: all 0.5s;
```

Comment:
It is better to add specific properties to the transition so that the browser does not watch to all properties, e.g.: `transition: color 0.3s ease, background-color 0.3s ease;`

Reply:
Agree with @grafit-off